app/policies/transaction_policy.rb

```diff
@@ -0,0 +1,11 @@
+class TransactionPolicy < ApplicationPolicy
```

Comment:
Should I add tests for this policies?

Reply:
@wazzuper yep, of course. Tests are vital